### PR TITLE
Do not load pretrained VGG

### PR DESCRIPTION
**Pull request by Yshuo-Li**

### CODE REVIEW

File: mmedit/models/extractors/lte.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __init__(self, requires_grad=True, pixel_range=1., pretrained=None):
             pixel_range=pixel_range, img_mean=vgg_mean, img_std=vgg_std)
 
         # use vgg19 weights to initialize
-        vgg_pretrained_features = models.vgg19(pretrained=True).features
+        vgg_pretrained_features = models.vgg19(pretrained=False).features
```
Comment (innerlee) [inline mmedit/models/extractors/lte.py:33]: What's the motivation of this change?

Comment (Yshuo-Li) [reply mmedit/models/extractors/lte.py]: Avoid downloading VGG in unittest of CI.

**Review by innerlee — APPROVED** [review]
